reference/5.1/Microsoft.PowerShell.Utility/Update-TypeData.md

```diff
@@ -3,131 +3,180 @@ external help file: Microsoft.PowerShell.Commands.Utility.dll-Help.xml
-To add the type data to all sessions, add an **Update-TypeData** command to your Windows PowerShell profile.
-For more information, see [about_Profiles](../Microsoft.PowerShell.Core/About/about_Profiles.md).
+Beginning in Windows PowerShell 3.0, you can use `Update-TypeData` to add and replace extended type
+data in the session without using a `Types.ps1xml` file. Type data that is added dynamically, that
+is, without a file, is added only to the current session. To add the type data to all sessions, add
+an `Update-TypeData` command to your PowerShell profile. For more information, see
+[about_Profiles](../Microsoft.PowerShell.Core/About/about_Profiles.md).
 
-Also, beginning in Windows PowerShell 3.0, you can use the Get-TypeData cmdlet to get the extended types in the current session and the Remove-TypeData cmdlet to delete extended types from the current session.
+Also, beginning in Windows PowerShell 3.0, you can use the `Get-TypeData` cmdlet to get the extended
+types in the current session and the `Remove-TypeData` cmdlet to delete extended types from the
+current session.
 
-Exceptions that occur in properties, or from adding properties to an **Update-TypeData** command, do not report errors to StdErr.
-This is to suppress exceptions that would occur in many common types during formatting and outputting.
-If you are getting .NET Framework properties, you can work around the suppression of exceptions by using method syntax instead, as shown in the following example:
+Exceptions that occur in properties, or from adding properties to an `Update-TypeData` command, do
+not report errors. This is to suppress exceptions that would occur in many common types during
+formatting and outputting. If you are getting .NET properties, you can work around the suppression
+of exceptions by using method syntax instead, as shown in the following example:
 
 `"hello".get_Length()`
 
-Note that method syntax can only be used with .NET Framework properties.
-Properties that are added by running the **Update-TypeData** cmdlet cannot use method syntax.
+Note that method syntax can only be used with .NET properties. Properties that are added by running
+the `Update-TypeData` cmdlet cannot use method syntax.
 
-For more information about the *types.ps1xml files in PowerShell, see [about_Types.ps1xml](../Microsoft.PowerShell.Core/About/about_Types.ps1xml.md).
+For more information about the `*types.ps1xml` files in PowerShell, see
+[about_Types.ps1xml](../Microsoft.PowerShell.Core/About/about_Types.ps1xml.md).
 
-## EXAMPLES
+## Examples
 
 ### Example 1: Update extended types
-```
-PS C:\> Update-TypeData
+
+```powershell
+Update-TypeData
 ```
 
-This command updates the extended type configuration from the *.types.ps1xml files that have already been used in the session.
+This command updates the extended type configuration from the `*.types.ps1xml` files that have already
```

Comment:
Another instance of referring to it slightly different.

Reply:
Fixed